composer/trainer/mosaic_fsdp_utils.py

```diff
@@ -687,6 +690,8 @@ def _custom_recursive_wrap_t2p1p0(
 
         module_kwargs = auto_wrap_policy(module=module, recurse=False, nonwrapped_numel=remainder)
         if not only_wrap_children and module_kwargs:
+            # CHANGE: We modify the original code to support custom FSDP kwargs and add
```

Comment:
Can you add a similar comment to these functions as well?

_custom_recursive_wrap_t1p13p1
custom_auto_wrap_t1p13p1
_custom_recursive_wrap_t2p0p1
init_fn_t2p0p1
_custom_auto_wrap_t2p0p1
_custom_auto_wrap_t2p1p0
init_fn_t2p1p0